osbenchmark/test_execution_orchestrator.py

```diff
@@ -185,7 +185,16 @@ def setup(self, sources=False):
         # to derive it ourselves. For source builds we always assume "master"
         if not sources and not self.cfg.exists("builder", "distribution.version"):
             distribution_version = builder.cluster_distribution_version(self.cfg)
-            self.logger.info("Automatically derived distribution version [%s]", distribution_version)
+            if distribution_version == 'oss':
+                self.logger.info("Automatically derived serverless collection, setting distribution version to 2.11.0")
+                distribution_version = '2.11.0'
```

Comment:
This might need to be updated in the future right? We should have a dedicated variable (`OSS_EQUIVALENT_DISTRIBUTION_VERSION`) somewhere where users can update in the future

Reply:
yes, 2.17 based release is in progress, good idea, will make this a constant.